
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_DEPORTS_AVEC_VARIATION_ENTRAXE(object):
    def setupUi(self, DEPORTS_AVEC_VARIATION_ENTRAXE):
        DEPORTS_AVEC_VARIATION_ENTRAXE.setObjectName("DEPORTS_AVEC_VARIATION_ENTRAXE")
        DEPORTS_AVEC_VARIATION_ENTRAXE.resize(1265, 419)

        DEPORTS_AVEC_VARIATION_ENTRAXE.setStyleSheet("background-color: #00aaff;")

        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("reducteur_gearexpert.gif"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        DEPORTS_AVEC_VARIATION_ENTRAXE.setWindowIcon(icon)
        self.gridLayout_2 = QtWidgets.QGridLayout(DEPORTS_AVEC_VARIATION_ENTRAXE)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.scrollArea = QtWidgets.QScrollArea(DEPORTS_AVEC_VARIATION_ENTRAXE)
        self.scrollArea.setWidgetResizable(True)
        self.scrollArea.setObjectName("scrollArea")
        self.scrollAreaWidgetContents = QtWidgets.QWidget()
        self.scrollAreaWidgetContents.setGeometry(QtCore.QRect(0, 0, 1251, 405))
        self.scrollAreaWidgetContents.setObjectName("scrollAreaWidgetContents")
        self.gridLayout_3 = QtWidgets.QGridLayout(self.scrollAreaWidgetContents)
        self.gridLayout_3.setObjectName("gridLayout_3")
        self.verticalLayout = QtWidgets.QVBoxLayout()
        self.verticalLayout.setObjectName("verticalLayout")
        self.gridLayout = QtWidgets.QGridLayout()
        self.gridLayout.setObjectName("gridLayout")
        self.pushButton_AVEC_VARIATION_6 = QtWidgets.QPushButton(self.scrollAreaWidgetContents)
        font = QtGui.QFont()
        font.setFamily("Arial Black")
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton_AVEC_VARIATION_6.setFont(font)
        self.pushButton_AVEC_VARIATION_6.setObjectName("pushButton_AVEC_VARIATION_6")

        self.pushButton_AVEC_VARIATION_6.setStyleSheet("background-color: #00ff7f; color: black;")

        self.gridLayout.addWidget(self.pushButton_AVEC_VARIATION_6, 4, 1, 1, 1)
        self.pushButton_AVEC_VARIATION_2 = QtWidgets.QPushButton(self.scrollAreaWidgetContents)
        font = QtGui.QFont()
        font.setFamily("Arial Black")
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton_AVEC_VARIATION_2.setFont(font)
        self.pushButton_AVEC_VARIATION_2.setObjectName("pushButton_AVEC_VARIATION_2")

        self.pushButton_AVEC_VARIATION_2.setStyleSheet("background-color: #00ff7f; color: black;")

        self.gridLayout.addWidget(self.pushButton_AVEC_VARIATION_2, 0, 1, 1, 1)
        self.pushButton_AVEC_VARIATION_3 = QtWidgets.QPushButton(self.scrollAreaWidgetContents)
        font = QtGui.QFont()
        font.setFamily("Arial Black")
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton_AVEC_VARIATION_3.setFont(font)
        self.pushButton_AVEC_VARIATION_3.setObjectName("pushButton_AVEC_VARIATION_3")

        self.pushButton_AVEC_VARIATION_3.setStyleSheet("background-color: #00ff7f; color: black;")

        self.gridLayout.addWidget(self.pushButton_AVEC_VARIATION_3, 2, 0, 1, 1)
        self.pushButton_AVEC_VARIATION_4 = QtWidgets.QPushButton(self.scrollAreaWidgetContents)
        font = QtGui.QFont()
        font.setFamily("Arial Black")
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton_AVEC_VARIATION_4.setFont(font)
        self.pushButton_AVEC_VARIATION_4.setObjectName("pushButton_AVEC_VARIATION_4")

        self.pushButton_AVEC_VARIATION_4.setStyleSheet("background-color: #00ff7f; color: black;")

        self.gridLayout.addWidget(self.pushButton_AVEC_VARIATION_4, 2, 1, 1, 1)
        self.pushButton_AVEC_VARIATION_5 = QtWidgets.QPushButton(self.scrollAreaWidgetContents)
        font = QtGui.QFont()
        font.setFamily("Arial Black")
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton_AVEC_VARIATION_5.setFont(font)
        self.pushButton_AVEC_VARIATION_5.setObjectName("pushButton_AVEC_VARIATION_5")

        self.pushButton_AVEC_VARIATION_5.setStyleSheet("background-color: #00ff7f; color: black;")

        self.gridLayout.addWidget(self.pushButton_AVEC_VARIATION_5, 4, 0, 1, 1)
        self.pushButton_AVEC_VARIATION = QtWidgets.QPushButton(self.scrollAreaWidgetContents)
        font = QtGui.QFont()
        font.setFamily("Arial Black")
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton_AVEC_VARIATION.setFont(font)
        self.pushButton_AVEC_VARIATION.setObjectName("pushButton_AVEC_VARIATION")

        self.pushButton_AVEC_VARIATION.setStyleSheet("background-color: #00ff7f; color: black;")

        self.gridLayout.addWidget(self.pushButton_AVEC_VARIATION, 0, 0, 1, 1)
        spacerItem = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout.addItem(spacerItem, 1, 0, 1, 1)
        spacerItem1 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout.addItem(spacerItem1, 3, 0, 1, 1)
        spacerItem2 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout.addItem(spacerItem2, 1, 1, 1, 1)
        spacerItem3 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout.addItem(spacerItem3, 3, 1, 1, 1)
        self.verticalLayout.addLayout(self.gridLayout)
        spacerItem4 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.verticalLayout.addItem(spacerItem4)
        spacerItem5 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.verticalLayout.addItem(spacerItem5)
        self.pushButton_AVEC_VARIATION_7 = QtWidgets.QPushButton(self.scrollAreaWidgetContents)
        font = QtGui.QFont()
        font.setFamily("Arial Black")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton_AVEC_VARIATION_7.setFont(font)
        self.pushButton_AVEC_VARIATION_7.setObjectName("pushButton_AVEC_VARIATION_7")

        self.pushButton_AVEC_VARIATION_7.setStyleSheet("background-color: #ff557f; color: black;")

        self.verticalLayout.addWidget(self.pushButton_AVEC_VARIATION_7, 0, QtCore.Qt.AlignHCenter)
        self.gridLayout_3.addLayout(self.verticalLayout, 0, 0, 1, 1)
        self.scrollArea.setWidget(self.scrollAreaWidgetContents)
        self.gridLayout_2.addWidget(self.scrollArea, 0, 0, 1, 1)

        self.retranslateUi(DEPORTS_AVEC_VARIATION_ENTRAXE)
        QtCore.QMetaObject.connectSlotsByName(DEPORTS_AVEC_VARIATION_ENTRAXE)

    def retranslateUi(self, DEPORTS_AVEC_VARIATION_ENTRAXE):
        _translate = QtCore.QCoreApplication.translate
        DEPORTS_AVEC_VARIATION_ENTRAXE.setWindowTitle(_translate("DEPORTS_AVEC_VARIATION_ENTRAXE", "DEPORTS_AVEC_VARIATION_ENTRAXE"))
        self.pushButton_AVEC_VARIATION_6.setText(_translate("DEPORTS_AVEC_VARIATION_ENTRAXE", "METHODE DE MAAG-TASCHENBUCH\n"
" POUR LA DENTURE HELICOIDALE "))
        self.pushButton_AVEC_VARIATION_2.setText(_translate("DEPORTS_AVEC_VARIATION_ENTRAXE", "METHODE ISO\n"
" POUR LA DENTURE HELICOIDALE"))
        self.pushButton_AVEC_VARIATION_3.setText(_translate("DEPORTS_AVEC_VARIATION_ENTRAXE", "METHODE DE W.RICHTER\n"
" POUR LA DENTURE DROITE "))
        self.pushButton_AVEC_VARIATION_4.setText(_translate("DEPORTS_AVEC_VARIATION_ENTRAXE", "METHODE DE W.RICHTER\n"
" POUR LA DENTURE HELICOIDALE "))
        self.pushButton_AVEC_VARIATION_5.setText(_translate("DEPORTS_AVEC_VARIATION_ENTRAXE", "METHODE DE MAAG-TASCHENBUCH\n"
" POUR LA DENTURE DROITE "))
        self.pushButton_AVEC_VARIATION.setText(_translate("DEPORTS_AVEC_VARIATION_ENTRAXE", "METHODE ISO\n"
" POUR LA DENTURE DROITE "))
        self.pushButton_AVEC_VARIATION_7.setText(_translate("DEPORTS_AVEC_VARIATION_ENTRAXE", "RETOUR"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    DEPORTS_AVEC_VARIATION_ENTRAXE = QtWidgets.QWidget()
    ui = Ui_DEPORTS_AVEC_VARIATION_ENTRAXE()
    ui.setupUi(DEPORTS_AVEC_VARIATION_ENTRAXE)
    DEPORTS_AVEC_VARIATION_ENTRAXE.show()
    sys.exit(app.exec_())
